public/doc/openapi-3-v3.0.json

```diff
@@ -1445,6 +1445,7 @@
             "type": "string"
           },
           "availability_status_error": {
+            "nullable": true,
```

Comment:
I don't think we need this @slemrmartin. Setting it to empty string should be fine.

Unless allowing it to be `nil` is better than `''` in your opinion?

Reply:
I didn't realize this is an option :) You're right